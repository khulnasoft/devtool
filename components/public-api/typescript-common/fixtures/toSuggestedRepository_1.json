{
  "url": "https://github.com/khulnasoft/devtool",
  "projectId": "123",
  "projectName": "Devtool",
  "repositoryName": "devtool"
}
