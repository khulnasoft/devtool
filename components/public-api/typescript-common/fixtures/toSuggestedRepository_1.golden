{
    "result": {
        "url": "https://github.com/khulnasoft/devtool",
        "repoName": "devtool",
        "configurationId": "123",
        "configurationName": "Devtool"
    },
    "err": ""
}
